modules/algebrasAndModel/src/main/scala/utils/TypeDecoders.scala

```diff
@@ -0,0 +1,21 @@
+package utils
+import cats.effect.Sync
+import io.circe.Decoder
+import org.http4s.circe._
+import utils.Types.{Organization, PostSubscriptionData, PostSubscriptions, Repository}
+import org.http4s.EntityDecoder
+import io.circe.generic.extras.semiauto.deriveUnwrappedDecoder
+import io.circe.generic.semiauto.deriveDecoder
+
+object TypeDecoders {
+  implicit val organizationDecoder: Decoder[Organization] = deriveUnwrappedDecoder[Organization]
+  implicit val repositoryDecoder: Decoder[Repository]     = deriveUnwrappedDecoder[Repository]
+  implicit val postSubscriptionsDataDecoder: Decoder[PostSubscriptionData] =
+    deriveDecoder[PostSubscriptionData]
+
+  implicit val postSubscriptionsDecoder: Decoder[PostSubscriptions] =
+    deriveDecoder[PostSubscriptions]
+
+  implicit def postSubscriptionsEntityDecoder[F[_]: Sync]: EntityDecoder[F, PostSubscriptions] =
+    jsonOf[F, PostSubscriptions]
```

Comment:
Since this is usually used and customized by the routes, I'd put this in the `Routes.scala`